app/src/sharedTest/java/org/oppia/android/app/policies/PoliciesFragmentTest.kt

```diff
@@ -325,6 +329,44 @@ class PoliciesFragmentTest {
     }
   }
 
+  @Test
+  fun testFragment_fragmentLoaded_verifyCorrectArgumentsPassed() {
+    launch<PoliciesFragmentTestActivity>(
+      createPoliciesFragmentTestIntent(
+        getApplicationContext(),
+        PolicyPage.TERMS_OF_SERVICE
+      )
+    ).use { scenario ->
+      testCoroutineDispatchers.runCurrent()
+      scenario.onActivity { activity ->
+
+        val policiesFragment = activity.supportFragmentManager
+          .findFragmentById(R.id.policies_fragment_placeholder) as PoliciesFragment
+
+        val policiesActivityParams = activity.intent.getProtoExtra(
+          PoliciesFragmentTestActivity.POLICIES_FRAGMENT_TEST_POLICY_PAGE_PARAMS_PROTO,
+          PoliciesActivityParams.getDefaultInstance()
+        )
+        val policiesFragmentArguments =
+          PoliciesFragmentArguments
+            .newBuilder()
+            .setPolicyPage(policiesActivityParams.policyPage)
+            .build()
+
+        val args = checkNotNull(policiesFragment.arguments) {
+          "Expected arguments to be passed to PoliciesFragment"
+        }
+        val receivedPolicies =
+          args.getProto(
+            "PoliciesFragment.policy_page",
```

Comment:
```suggestion
            PoliciesFragmentTestActivity.POLICIES_FRAGMENT_TEST_POLICY_PAGE_PARAMS_PROTO,
```

Reply:
I think , we should use  PoliciesFragment.POLICIES_FRAGMENT_POLICY_PAGE_ARGUMENT_PROTO.
to use it , we need to set POLICIES_FRAGMENT_POLICY_PAGE_ARGUMENT_PROTO as "Public" in PoliciesFragment.


If we use PoliciesFragmentTestActivity.POLICIES_FRAGMENT_TEST_POLICY_PAGE_PARAMS_PROTO. we are getting argument of PoliciesFragmentTestActivity and the test is failed.

Could you please review it once and confirm what should i use?

Reply:
You are right, it should be `PoliciesFragment.POLICIES_FRAGMENT_POLICY_PAGE_ARGUMENT_PROTO`. Please update it accordingly.

Reply:
Okay, fine. I'll make the update accordingly.

Reply:
Done.